### ISSUE TITLE
Overfitting on ResNet18

### ISSUE THREAD

**Issue by MichaelLee-ceo** [kuangliu/pytorch-cifar#161]
I copied the resnet.py for constructing ResNet18 and start training. 
The hyperparameters I used to train the network are the same as defined in the main.py, but I eventually end up with overfitting on the testing set.
I got almost 98% accuracy on training set while only 89% on testing set.
Did I mis-configured something or how can I handle it to avoid overfitting? Thanks

**Comment by kwang3939** [comment]
Same here. I also got about 85% on the test set with ResNet 18. I haven't figured out the reason yet. 
Have you addressed this issue? Thanks

**Comment by MichaelLee-ceo** [comment]
Hi, I have figured out that the learning rate scheduler "CosineAnnealingLR" has an impact on both training and testing accuracy, which can make the model more converged at the end of the training.
After I added it back to my code, I can achieve about 93% on ResNet18, so maybe you try that.
